pages/studentPortal/labs/multiplication-connect/Index.tsx

```diff
@@ -1,16 +1,42 @@
-import React from 'react'
+import React, { useState } from 'react'
 import DiceSection from '../../../../components/math/multiplicationConnect/DiceSection';
 import GameBoard from '../../../../components/math/multiplicationConnect/GameBoard';
 import PlayerSection from '../../../../components/math/multiplicationConnect/PlayerSection';
+import { getRandomItemFromArray } from '../../../api/random';
+
+const calculateWinner = () => {
```

Comment:
why do you have an empty function? should we remove it?

Reply:
Yeah I didn't intend for this to be in this PR. Should've created a new commit.